app/models/portal/learner.rb

```diff
@@ -1,5 +1,9 @@
 class Portal::Learner < ActiveRecord::Base
+  include Rails.application.routes.url_helpers
+
   self.table_name = :portal_learners
+
+  KEY_PREFIX = 'key:'
```

Comment:
It would be okay, but as we are removing path filtering in LARA, I would just remove this prefix at all.